src/main/kotlin/org/b12x/gfe/core/model/DataDownload.kt

```diff
@@ -0,0 +1,63 @@
+package org.b12x.gfe.core.model
+
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import okhttp3.Response
+import org.b12x.gfe.core.model.parsers.ParserVersionData
+
+/**
+ * Downloads data as indicated
+ *
+ * @param loci what group of genes you are downloading data for
+ */
+class DataDownload(lociGroup: String) {
```

Comment:
This looks like a static utility class despite nominally having a constructor and parameter. Consider rewriting Kotlin-style (https://proandroiddev.com/utils-class-in-kotlin-387a09b8d495) and adding a parameter to `parseResponse()`.

Reply:
That makes sense (I have a few other objects floating around so that's no problem). This is another incomplete class as once again, they haven't finished the API so I don't know what will be required to make the data request. This will probably change, and I will save this comment for when I work on it again.